app/config/start.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Scripts that run the command: sct_label_utils -i <input file> -create-viewer integer1,integer2,integerN
+# Makes sure that <input file> and integer1,integer2,integerN are provided in the input ports
+
+set -e
+
+INFO="INFO: [$(basename "$0")] "
+
+while [ ! -f ${DY_SIDECAR_PATH_INPUTS}/input_1/input_image.nii.gz ]
+    do 
+    sleep 1
+    echo "$INFO: Waiting for image file in first input port"
+    done
+
+while [ ! -f ${DY_SIDECAR_PATH_INPUTS}/input_2/labels.txt ]
+    do 
+    sleep 1
+    echo "$INFO: Waiting for input labels txt file in second input port"
+    done
```

Comment:
In the future you might also want to validate the content of this file. That way you can hint users on what is happening.

Reply:
Indeed. In the future I would like to have this second input provided as a list of integers in the input port (and validated by the content schema, I have a TODO for that in .osparc/web/metadata.yml)